{"name": "test_getbyname_failure", "status": "passed", "start": 1729079583592, "stop": 1729079583593, "uuid": "9be6a322-bcf3-4a3a-9c08-ef3bc84f8205", "historyId": "a93088beb91554d7a6f791d56bf46900", "testCaseId": "a93088beb91554d7a6f791d56bf46900", "fullName": "Exercise.exercise_test.TestExerciseServiceWithoutMock#test_getbyname_failure", "labels": [{"name": "parentSuite", "value": "Exercise"}, {"name": "suite", "value": "exercise_test"}, {"name": "subSuite", "value": "TestExerciseServiceWithoutMock"}, {"name": "host", "value": "DESKTOP-IDGJ8R7"}, {"name": "thread", "value": "1956-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "Exercise.exercise_test"}]}